external/js/cothority/src/byzcoin/proto/requests.ts

```diff
@@ -1,3 +1,4 @@
+import Long from "long";
```

Comment:
you can remove it

Reply:
I get the following tslint error if I remove the import:
```
        const signer = SignerEd25519.fromBytes(ephemeral);
```
But I'm not sure if my tslint is badly configured.